pkg/core/localize/locales/en/cmd/connectors.toml

```diff
@@ -0,0 +1,261 @@
+["connector.addon.params.cmd.shortDescription"]
```

Comment:
Caused by the bash script
`["connector.addon.params.cmd.shortDescription"]` => `[connector.addon.params.cmd.shortDescription]`